utide/confidence.py

```diff
@@ -252,7 +252,10 @@ def _confidence(coef, cnstit, opt, t, e, tin, elor, xraw, xmod, W, m, B,
             varcov_mCw[c, :2, :2] = Duu
 
             if not opt.white:
-                Duu = Puu[c] * Duu / np.trace(Duu)
+                Duu = Puu[c] * Duu
+                trace = np.trace(Duu)
```

Comment:
This is not the same as the original even in the normal case.  The order of the two lines above would need to be swapped.